resolver/resolver.cc

```diff
@@ -108,6 +108,14 @@ class ResolveConstantsWalk {
 
         AncestorResolutionItem(const AncestorResolutionItem &rhs) = delete;
         const AncestorResolutionItem &operator=(const AncestorResolutionItem &rhs) = delete;
+
+        inline core::SymbolRef symbolForStubbing() const {
```

Comment:
could you unmark it as `inline` please? 
Without strong guidance, we try to not abuse optimize hints.
Additionally, could this be a method that takes `AncestorResolutionItem` instead  of being a class member?
So far `AncestorResolutionItem` doesn't have any methods.